docs/content/advanced/backend/cron-jobs/create.md

```diff
@@ -51,9 +51,15 @@ const publishJob = async (container, options) => {
 export default publishJob;
 ```
 
-This file should export a function that accepts a `container` and `options` parameters. `container` is the dependency container that you can use to resolve services, such as the [EventBusService](../../../references/services/classes/EventBusService.md). `options` are the plugin’s options if this cron job is created in a plugin.
+:::info
+
+The service taking care of background jobs was renamed in 1.7.1. If you are running a version < 1.7.1, use `eventBusService` instead of `backgroundJobService`.
```

Comment:
```suggestion
The service taking care of background jobs was renamed in v1.7.1. If you are running a previous version, use `eventBusService` instead of `backgroundJobService`.
```

Reply:
I like it. I'll then proceed with renaming the file as well 👍